design/video-infrastructure.md

```diff
@@ -2,49 +2,49 @@
-We can also decrease the image quality for viewers with slower connections or smaller displays. If the viewer has a slower connection we still want to provide them a stream to watch, and to do this we need to be able to provide a lower quality stream for them. Or if they have a smaller display providing a higher resolution stream does not make any difference and can in some cases look worse.
+Transcoding is crucial since we can significantly reduce the bandwidth cost of the video by compressing it using codecs such as [H264](https://en.wikipedia.org/wiki/Advanced_Video_Coding), [VP8](https://en.wikipedia.org/wiki/VP8), [VP9](https://en.wikipedia.org/wiki/VP9) or [AV1](https://en.wikipedia.org/wiki/AV1). 
+We can also decrease the image quality for viewers with slower connections or smaller displays. If the viewer has a slower connection, we still want to provide them with a stream to watch and to do this. We need to be able to provide a lower-quality stream for them. Or if they have a smaller display providing a higher resolution stream does not make any difference and can sometimes look worse.
 
 ### Video Codecs
 
-There are a few video codes mentioned in the above paragraph. Generally H264 has been the golden standard for most if not all of live video for a very long time, due to the large amount of client support. However AV1 provides much better quality per bit compression. On average it results in about 50% smaller files. So a 8k bitrate in H264 would be a 4k bitrate stream in AV1. This allows us to ship higher quality video to viewers with slower internet speeds and also allows us to save a lot of money in CDN caching and bandwidth costs. However AV1 is much harder to transcode than H264. So transcoding becomes a higher cost.  
+There are a few video codes mentioned in the above paragraph. Generally, H264 has been the golden standard for most, if not all, of live video for a very long time due to a large amount of client support. However, AV1 provides much better quality per-bit compression. On average, it results in about 50% smaller files. So an 8k bitrate in H264 would be a 4k bitrate stream in AV1. This allows us to ship higher-quality video to viewers with slower internet speeds and also allows us to save a lot of money in CDN caching and bandwidth costs. However, AV1 is much harder to transcode than H264. So transcoding becomes a higher cost.
 
 ## Edge
 
-We are going to have many servers in multiple regions around the world like Europe, North America, South America, South East Asia, Australia, Africa.
+We will have many servers in multiple regions worldwide, like Europe, North America, South America, South East Asia, Australia, and Africa.
```

Comment:
Not sure "worldwide" is the right word here, "around the world" felt more natural to me :shrug:

Reply:
Worldwide is fine